additional-lambdas/cost_metrics/cost_metrics.js

```diff
@@ -49,60 +53,68 @@ function find_file_in_s3(s3) {
           for (var i = 0; i < data.Contents.length; i++) {
             var name = data.Contents[i].Key;
             if (re.test(name)) {
-                if (latestobj.length === 0) { latestobj.push(data.Contents[i]); }
+                if (latestObj.length === 0) { latestObj.push(data.Contents[i]); }
                 else {
-                if (new Date(latestobj[0].LastModified).getTime() < new Date(data.Contents[i].LastModified).getTime()){
-                    latestobj.pop();
-                    latestobj.push(data.Contents[i]);
-                    console.log("latest: " + JSON.stringify(latestobj));
+                if (new Date(latestObj[0].LastModified).getTime() < new Date(data.Contents[i].LastModified).getTime()){
+                    latestObj.pop();
```

Comment:
Created card [AUTO-962](https://gaautobots.atlassian.net/browse/AUTO-962) for this.